buildpack.md

```diff
@@ -4,17 +4,21 @@ This document specifies the interface between a lifecycle program and one or mor
 
 The lifecycle program uses buildpacks to build software artifacts from source code and pack the result into an OCI image.
 
-This is accomplished in four phases:
+This is accomplished in six phases:
 
-1. **Detection,** where an optimal selection of compatible buildpacks is chosen.
-2. **Analysis,** where metadata about OCI layers generated during a previous build are made available to buildpacks.
-3. **Build,** where buildpacks use that metadata to generate only the OCI layers that need to be replaced.
-4. **Export,** where the remote layers are replaced by the generated layers.
+1. **Analysis**, where metadata about OCI layers generated during a previous build are made available to buildpacks.
+2. **Detection**, where an optimal selection of compatible buildpacks is chosen.
+3. **Build-Ext** (optional, see the [Image Extension Interface](image-extension.md)), where image extensions generate Dockerfiles that can be used to extend the build and/or run base images.
```

Comment:
`Build-Ext` is not the best name... I like `Generate` because the purpose of this phase is to **generate** Dockerfiles.

Reply:
I like "Generation"

Reply:
If we call it Generation, then having the extension script become /bin/generate (rather than /bin/build) would be nice & consistent.

Reply:
Updated in bf837f6